pkgs/development/libraries/science/math/cudnn/extension.nix

```diff
@@ -5,7 +5,11 @@ final: prev: let
 
   ### CuDNN
 
-  buildCuDnnPackage = args: callPackage ./generic.nix {} args;
+  buildCuDnnPackage = args:
+    let
+      useCudatoolkitRunfile = lib.versionOlder args.fullVersion "11.4.0";
```

Comment:
I'd move it to `args`, but I'd have to split `8.3.2.44` into two instances - pre and post cuda 11.4 - and adjust the selector expression smh... I'll keep it this way, I guess